concepticondata/concepticon.tsv

```diff
@@ -1782,12 +1782,12 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 1781	THUMB	The body	The shortest and thickest of the fingers that can be placed against the others in order to create a firm grip.	Person/Thing	
 1782	GUEST	Social and political relations	A person that visits and stays at a house, usually for one or more nights.	Person/Thing	
 1783	PAIN	Emotions and values	An unpleasant, usually localised physical sensation that is often the result of an injury, disease or other ailment.	Person/Thing	
-1784	WANT	Emotions and values	To have a strong desire for something.	Action/Process	
+1784	WANT	Emotions and values	To want to do something or to wish for something to happen.	Action/Process	
 1785	HAVE SEX	The body	To take part in sexual activity (most commonly sexual intercourse) with another person for the purposes of sexual pleasure.	Action/Process	549
 1786	DRY UP	Basic actions and technology	To lose moisture, usually through evaporation or absorption.	Action/Process	
 1787	LOOM	Clothing and grooming	A frame or machine (made of wood or other material) in which a weaver forms cloth out of thread; a machine for interweaving yarn or threads into a fabric.	Person/Thing	
 1788	TELEVISION	Modern world	A device for receiving television signals and displaying them in visual form.	Person/Thing	
-1789	AGE	Time	The period of time that a person, animal or plant has lived or is expected to live.	Person/Thing	
+1789	AGE	Time	The amount of time that a person, animal or plant has lived.	Person/Thing	
```

Comment:
Or that an object came into being (but not sure if this should be added to the definition).